{{cookiecutter.project_slug}}/README.md

```diff
@@ -3,10 +3,26 @@
 {{ cookiecutter.package_description }}
 
 ### Setup
-
+{% if cookiecutter.visibility == 'public' %}
 ```bash
 pip install {{ cookiecutter.project_slug }}
 ```
+{% else %}
+We install private packages directly via GitHub.
+
+For pip simply add
+
+```
+-e git+https://GITHUB_TOKEN@github.com/Thermondo/REPO_NAME.git@VERSION_TAG#egg=PACKAGE_NAME
```

Comment:
You can replace `REPO_NAME` and `PACKAGE_NAME` with `{{ cookiecutter.project_slug }}`